src/Daemon/SentryWriter.cpp

```diff
@@ -79,17 +79,9 @@ void SentryWriter::resetInstance()
 
 SentryWriter::SentryWriter(Poco::Util::LayeredConfiguration & config)
 {
-    bool enabled = false;
-    bool debug = config.getBool("send_crash_reports.debug", false);
     auto logger = getLogger("SentryWriter");
 
-    if (config.getBool("send_crash_reports.enabled", false))
```

Comment:
I think we should keep this check as otherwise we'll send to sentry tons of local dev crashes. Better to only send CI generated builds.

Reply:
I don't mind receiving a ton of reports. Moreover, I'd like to see them as proof that everything works. We can easily filter them, as it also sends the "official" flag.